woocommerce/woocommerce-gutenberg-products-block
# Move Button, StoreNotice and StoreNoticesContainer components into the components package

## What
This PR moves the Button, StoreNotice and StoreNoticesContainer components into the components package

Fixes #11685
Fixes #11683

## Why
We are consolidating all the components that we will export for external use into a `components` package



## Testing Instructions



_Please consider any edge cases this change may have, and also other areas of the product this may impact._

1. Install the [WooCommerce Stripe Payment Gateway](https://wordpress.org/plugins/woocommerce-gateway-stripe/)
2. Add something to your cart
3. Visit the Cart & Checkout and make sure all the buttons appear and function as expected
5. Try to make a payment with a random stripe card (or an known invalid one).
6. You should see a notice appear in the payment section of the checkout

## Internal Developer Testing (do not include in the notes)

1. Go to the cart page and enter the following in console: 

`let { createErrorNotice } = wp.data.dispatch('core/notices'); createErrorNotice('Hello there', { context: 'wc/cart' });`

3. You should see a notice saying "Hello there".
4. Dismiss this notice and make sure it disappears
5. Go to the checkout page
6. In the console, type each of the following. They should all be functions
 - window.wc.blocksCheckout.Button
 - window.wc.blocksCheckout.StoreNotice
 - window.wc.blocksCheckout.StoreNoticesContainer
 - window.wc.blocksComponents.Button
 - window.wc.blocksComponents.StoreNotice
 - window.wc.blocksComponents.StoreNoticesContainer




## WooCommerce Visibility


Required:


## Checklist

Required:


Conditional:


## Changelog


> We have moved the Banner, StoreNotice and StoreNotices components to the @woocommerce/blocks-components package. Previously these were available in @woocommerce/blocks-checkout . Your code will continue to work as we have added aliases to the new location. Despite this, it is recommended that you change your code to import this component from @woocommerce/blocks-components as the import from the checkout package will be deprecated in the future.
